samples/bot-ai-meeting-helper/README.md

```diff
@@ -0,0 +1,186 @@
+---
+page_type: sample
+description: This sample showcases how to utilize Azure Open AI Search within Teams Toolkit for Visual Studio Code to extract Action Items and Meeting summaries from meeting transcriptions.
+products:
+- office-teams
+- copilot-m365
+languages:
+- javascript
+---
+
+# Meeting Helper with Azure Open AI
+
+This sample demonstrates generating action items and a meeting summary based on the attendees and transcription, then sending them to all participants.
+
+- To achieve this process, the user will schedule a meeting, which can be either a one-time or recurring event. Attendees interested in receiving the meeting summary and action items must individually subscribe to the meeting. For recurring meetings, multiple instances will be displayed, allowing users to subscribe to each instance separately.
+ 
+- After the meeting concludes, a webhook endpoint will be triggered in the background to extract the meeting transcription.
+
+- Using Azure OpenAI, the extracted meeting transcription will be processed to extract action items and a meeting summary. Finally, this information will be sent to all meeting attendees who have subscribed to the meeting.
+
+## Included Features
+* **Graph API:** Utilize graph APIs to retrieve meeting transcriptions and other meeting or user-related details.
+* **Azure Open AI:** Leverages Azure OpenAI for extracting action items and meeting summaries.
+* **Azure Table Storage:** Utilizes Azure Table Storage to store subscribed user information, including conversation IDs for sending meeting details.
+* **Self-Signed Certificate:** Uses Azure self-signed certificates for encryption and decryption to establish the meeting subscription.
+
+## Interaction with app
+
+ ![bot-ai-meeting-helperGif](Images/bot-ai-meeting-helperGif.gif)
+
+## Prerequisites
+
+- [Node.js 18.x](https://nodejs.org/download/release/v18.18.2/)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Table Storage](https://learn.microsoft.com/en-us/azure/storage/tables/table-storage-quickstart-portal)
+- [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension)
+- [Azure OpenAI](https://learn.microsoft.com/en-us/azure/ai-services/openai/quickstart?tabs=command-line&pivots=programming-language-studio)
+- [PowerShell](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-macos) if on macOS (`brew install --cask powershell`)
+- [Self-Signed Certificate](https://learn.microsoft.com/en-us/azure/key-vault/certificates/quick-create-portal)
+
+### Create an Azure Open AI service
+- In Azure portal, create a [Azure Open AI service](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource?pivots=web-portal).
```

Comment:
Which AI Model we need to deploy?

It should be mentioned here.

Reply:
Refer this PR:
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/1335/files#diff-820624b3d7f7e4be48d0fc48df9ab543ee07089829a4bf35bd2e808c12637bb9R42

Reply:
Done